frontend/src/components/HeroSection.tsx

```
@@ -0,0 +1,37 @@
+      <div className="flex flex-col justify-center items-center gap-8 w-[660px]">
+        <div className="flex flex-col justify-center items-center gap-4 w-full">
+          <h1 className="font-roboto font-medium text-[57px] leading-[64px] tracking-[-0.25px] text-center text-black">
+            {t('hero.title')}
+          </h1>
+
+          <p className="font-roboto font-normal text-[28px] leading-[36px] text-center text-black">
+            {t('hero.description')}
+          </p>
+        </div>
+
+        <button className="flex flex-row justify-center items-center w-[203px] h-[48px]">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)